shared/wallets/wallet/settings/index.tsx

```diff
@@ -35,6 +35,12 @@ export type SettingsProps = {
   thisDeviceIsLockedOut: boolean
 }
 
+const HoverBox = Styles.isMobile
+  ? Kb.ClickableBox
+  : Styles.styled(Kb.ClickableBox)({
```

Comment:
just use `className='hover-underline'` and .hover-underline-container (see style.css)